xocto/storage/storage.py

```diff
@@ -701,7 +701,7 @@ def get_size_in_bytes(self, *, s3_object: S3Object) -> int:
         boto_object = self._get_boto_object(s3_object=s3_object)
         return boto_object.content_length
 
-    def get_last_modified(self, *, s3_object: S3Object) -> datetime:
+    def get_last_modified(self, *, s3_object: S3Object) -> datetime.datetime:
```

Comment:
This should be in a fixup commit and rebased into the commit that added this to avoid non-atomic commits where the type checker is broken.